contents/page-sections/authentication/forgot-password/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { ArrowLeft } from "@yamada-ui/lucide"
+import { ArrowLeft as ArrowLeftIcon } from "@yamada-ui/lucide"
```

Comment:
`ArrowLeftIcon` can be imported without aliasing.

```suggestion
import { ArrowLeftIcon } from "@yamada-ui/lucide"
```